unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -5011,10 +5018,12 @@ private IEnumerator checkDropHandObjectActionFast(SimObjPhysics currentHandSimOb
 
             GameObject go = ItemInHand;
             DropHandObject(action);
-            if (this.lastActionSuccess) {
+            // Force is not applied because action success from DropHandObject starts a coroutine that waits for the object to be stationary
+            // to return lastActionSuccess == true that is not what we want for throwing an object, review why this was that way
+            //if (this.lastActionSuccess) {
```

Comment:
@winthos just your input for this

Reply:
Yeah I think we need to determine what counts as the action successfully completing. Is it when the object leaves the hand and is no longer being held, or is it when it comes to rest after having been to thrown?

I’m not sure which makes the most sense